client/thin-replica-client/include/client/thin-replica-client/thin_replica_client.hpp

```diff
@@ -164,6 +166,10 @@ struct ThinReplicaClientMetrics {
   // the TRC, to when it is pushed to the update queue for consumption by the
   // application using TRC
   concordMetrics::GaugeHandle update_dur_ms;
+  // number of updates received by TRC per second
+  concordMetrics::GaugeHandle updates_received_per_second;
```

Comment:
Specifically, this is the number of data updates (no hash updates) per second and they could come from different replicas. However, it doesn't tell us how many updates we actually processed which is why we have the other value below. So, I'm pondering how do we use this value during analysis? I guess, this is only interesting if the two `updates_received` and `updates_processed` diverge but the system doesn't get stuck. Then we can conclude that we find an f+1 agreement amongst all replicas but we get "unlucky" in how we choose the data stream!?

Reply:
I agree, I have removed `updates_received_per_second`.